package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "@parcel/source-map",
-  "version": "2.0.0-alpha.4.3",
+  "version": "2.0.0-alpha.4.4",
```

Comment:
No longer have access to master so this is the only real way I can trigger a release